meta-freescale-3rdparty-mbl/conf/machine/imx7d-pico-mbl.conf

```diff
@@ -126,3 +126,5 @@ MBL_WKS_BOOTLOADER2_IS_BL3 = "1"
 
 # As reported by "mmc extcsd read /dev/mmcblk1"
 MBL_FLASH_ERASE_BLOCK_SIZE_KiB = "512"
+
+MBL_WKS_STORAGE_SIZE_MiB = "7373"
```

Comment:
Should we have a comment here explaining where the extra 205MiB comes from compared to most of the other boards? Maybe this is obvious but I don't understand why there's a difference.

Reply:
I think we should have a comment about the storage size for all machines.

Reply:
Yes, we should. My lack of comments was also hiding my sloppy process for coming up with the numbers...

Reply:
Fixed in https://github.com/ARMmbed/meta-mbl/pull/674/commits/7041a0d1661c98432bf2efb4e4e2abaa7ec32222.